data.sql

```diff
@@ -53,3 +53,134 @@ SET owner_id =
         WHEN name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom' THEN (SELECT id FROM owners WHERE full_name = 'Melody Pond')
         WHEN name = 'Angemon' OR name = 'Boarmon' THEN (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
     END;
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you put your own name and logo created by you at the top of your readme file to present yourself as a brand in the Job Market and Tech industry to catch the recruiters' eyes 👍